src/main.ts

```diff
@@ -35,7 +33,6 @@ async function start() {
 	// We are always behind a proxy, but we want the source IP
 	probot.server.set("trust proxy", true);
 	probot.load(App);
-	probot.webhook.on("error", (err: Error) => logger.error({ ...err }, "Webhook Error"));
```

Comment:
This lead to event payload being logged.

Without this line probot logs an error anyway